cypher/cypher-docs/src/docs/dev/ql/administration/security/deny-match-syntax.asciidoc

```diff
@@ -2,11 +2,12 @@
 [source, cypher]
 -----
 DENY MATCH
-    { * | properties }
+    "{" { * | properties } "}"
```

Comment:
@OliviaYtterbrink should this be `"{" { * | property[, ...] } "}"`? (probably a bit of everywhere and multiple versions so I guess I should make a card for it)

Reply:
I guess same goes for `roles` -> `role[, ...]`

Reply:
Just make sure we actually support multiples of all of them in 4.0....

Reply:
For properties and roles I'd say we do, don't think those were ever singular, but yes would be good to confirm (and see if there are more versions like this)

Reply:
Added a card, https://trello.com/c/nZWT53fV/2265-graph-privileges-comma-separated-lists